### PR TITLE
Fix cpu_percent, when statsinterval > 1 

### PR DESCRIPTION
**Pull request by megastef**
Discoverd a lot of 0 cpu values when statsinterval was > 1.
The reason was a negative delta in cpu - prevCpu

### CODE REVIEW

File: stats.js [hunk 1]
```diff
@@ -81,8 +81,8 @@ function stats(opts) {
             stats: stats
           })
 
-          previousCpu = cpuSum
-          previousSystem = sysSum
+          previousCpu = stats.cpu_stats.cpu_usage.total_usage
+          previousSystem = stats.cpu_stats.system_cpu_usage
```
Comment (mcollina) [inline stats.js:85]: why these changes?

Comment (megastef) [inline stats.js:85]: I've started to use a statsinterval>1 (e.g. 10 - to reduce load on the monitoring agent) and got always 0.0 as value for cpu_percent for some 'sleeping' containers. Then I debugged and discovered that previousCpu was higher than cpuSum and the difference got a negative value. There is a if condition that returns then 0.0 when negative - explaining why I got always 0.0. cpu percent. And I wondered how this could happen. After making this change my cpu value looked correct like 0.38% - the same value I had with statsinterval=1.

**Comment by mcollina** [comment]
Released as 0.5.1.

**Comment by megastef** [comment]
Thx.
